### DISCUSSION TITLE
Recomendação - UVA x Cruzeiro do sul

### DISCUSSION THREAD

**Discussion by KumoMg** [frontendbr/forum#1462] [Discussão]
Bom dia pessoal, gostaria de saber se alguem já fez Analise de desenvolvimento de sistemas (ead) na Universidade Veiga de Almeida ou na Universidade Cruzeiro do Sul? Se sim, você poderia falar um pouco da sua opinião sobre a faculdade e o curso?

O preço das duas está bem similar, o que me atraiu na Cuzeiro foi o fato de durar apenas 4 semestes.

**Comment by DiogoCercilio** [comment]
Cara, quer um conselho de verdade mesmo? Fuja da Cruzeiro do Sul... mas fuja muuuuito mesmo. É a pior faculdade que existe no mundo cara. Na verdade, é a pior instituição de ensino de qualquer coisa que eu já vi. Vou citar apenas alguns dos exemplos do que aconteceu comigo, porque se for falar tudo vai ficar muito extenso:

- em 80% do tempo, a área do aluno está fora (não é exagero). Quase sempre é erro 500;
- Não importa se a área do aluno está fora o tempo todo, o prazo se mantém;
- Quando consegue se conectar (geralmente de madrugada), as vídeo aulas são textos nada didáticos, sobre assuntos extremamente rasos, e lidos por um professor que parece um robô lendo;
- As provas online tem várias respostas erradas. 99% dos alunos reclamam no fórum, mas a faculdade sequer nos respondem, e tacam nota negativa;
- Quando precisar resolver algo presencialmente na faculdade, eles vão te dizer que não podem fazer nada, que só resolvem abrindo chamado na plataforma;
- A plataforma não vai funcionar quando você for abrir o chamado;
- No dia da prova, a plataforma não vai funcionar, você vai tentar reclamar, e eles não vão nem te ouvir;
- Vão fazer cobranças indevidas, e você não vai conseguir reclamar com ninguém... Se conseguir abrir um chamado, eles vão encerrar sem mais nem menos.

Acho que já falei muito... Não conheço a outra, mas não faz Cruzeiro do Sul cara, o barato vai sair caro. Você vai pegar várias DPs por não conseguir acessar a plataforma.

Espero ter ajudado,
Abraço!

**Comment by Juuhz** [comment]
Eu comecei minha graduação em analise e desenvolvimento de sistemas já tendo curso técnico ( focado em desenvolvimento, onde fiquei 1 ano e meio ) e 1 ano de mercado de trabalho. 

Ou seja, já comecei minha faculdade tendo experiência em desenvolvimento, então eu queria apenas o diploma para almejar cargos maiores. 

Optei por fazer presencial, oque foi uma decisão bem ruim, pois eu trabalhava 8 horas por dia e ainda tinha que ir para faculdade, oque me sugava muito, minha sorte é que, por eu ter experiência na área, minhas notas não eram tão ruins. 

No meu penúltimo período minha faculdade fechou, então por já ter uma experiência ruim em presencial, optei por continuar meu curso em outra instituição EAD ( melhor decisão que fiz haha ). 

Cara, é bem tranquilo, veja os pontos positivos:

- A média é menor.
- Você tem tempo.
- Não precisa se preocupar com faltas/presença. 
- As provas são fim de semana.
- Você vai descansado para as provas.
- O período para realização das provas são de 1 mês ( pelo menos na Estacio, onde fiz ). 
- Você marca as provas para os dias que você quiser. 
- Não tem professor biscoito. 
- O diploma é o mesmo da presencial.

Sobre a área do aluno ( lugar onde você faz trabalhos e estuda ), no geral, é bem ruim, coisas feitas em linguagens antigas e costuma sair do ar com frequência, ( conforme o @DiogoCercilio mencionou ). 

Mas no meu caso, como eu queria apenas fazer a receitinha de bolo para pegar meu diploma ( fazer os trabalho 1 semana antes, ler o conteúdo no dia da prova ), não me impactou negativamente.

O ruim do EAD é a auto-gestão, se você for um cara que já sabe ( tipo eu ), você tira de letra, mas se você tiver com intuito de aprender a programar na Faculdade ( oque julgo burrice ), faz presencial que é melhor. 

Espero ter ajudado. 

Abraços e boa sorte.
